SharedUtils/src/main/java/com/braintreepayments/api/SynchronousHttpClient.java

```diff
@@ -73,8 +73,8 @@ String request(HttpRequest httpRequest) throws Exception {
             httpRequest.dispose();
         }
 
-        int responseCode = connection.getResponseCode();
         try {
+            int responseCode = connection.getResponseCode();
```

Comment:
This change fixes an integration test where we are testing that an invalid endpoint returns an error. Currently no error is returned because no connection is ever made. I think this update is the intended behavior (returns error on no connection).